test/unit/util/handlers/ProcessHandler.test.ts

```diff
@@ -0,0 +1,90 @@
+import cluster from 'cluster';
+import type { AsyncHandler } from '../../../../src';
+import { NotImplementedHttpError, ClusterManager, ProcessHandler } from '../../../../src';
+
+jest.mock('cluster');
+
+function setPrimary(primary: boolean): void {
+  const mockCluster = jest.requireMock('cluster');
+  const props = { isMaster: primary, isWorker: !primary };
+  Object.assign(mockCluster, props);
+}
+
+jest.mock('../../../../src/init/ClusterManager', (): any => ({
```

Comment:
`ProcessHandler` does not depend on the implementation of `ClusterManager` so there's no need to mock it like this, it only depends on its types. So the object should be made in the same way as for example the `source` object below.